### ISSUE TITLE
Jolly confusing initial screen

### ISSUE THREAD

**Issue by LinuxOnTheDesktop** [qTox/qTox#6100]
##### Brief Description

OS: Windows / OS X / Linux (include version and/or distro)  Mint 19.3 Cinnamon x64
qTox version: (version numbers in menu `Settings → About`): v1.16.3-888-g0f338f74. (compiled from source, because qTox is not in my repos)

Commit hash:  0f338f749ca7ed5b90e1aacbd319df86719be12b
toxcore:  0.2.11
Qt:  5.9.5
…

**Reproducible**: Always

##### Steps to reproduce

1. Install.
2. Run.

##### Observed Behavior

A baffling screen - see my screenshot below.

##### Expected Behavior

I expected something that I, and my less tech savvy friends, can understand. Where do I get that 'Tox ID' from? Once I have it, what exactly am I doing with it? I presume - after some thought - that the ID would act as an address to which I send the friend request. Is this right? If it is (and even if is not) where do I get that ID?

##### Additional Info

![image](https://user-images.githubusercontent.com/47056543/79683214-6697a500-8220-11ea-8b53-ec14b4cc6c93.png)

**Comment by Chiitoo** [comment]
Indeed, the Tox ID would be the ID used to connect with other users.

To find their ID, a user can click on their avatar or/and nick on the main window.

See also:

- https://github.com/qTox/qTox/issues/2657
- https://github.com/qTox/qTox/blob/master/doc/user_manual_en.md

**Comment by anthonybilinski** [comment]
I agree this needs work. Clicking the avatar to find your profile when the rest of the settings are all bottom left is also confusing.

I think both making the inital UI more clear (finding your own Tox ID, having it more clear why you would need it, more clear how to add people), as well as having first time help popups would be good to add.

**Comment by jessica181920** [comment]
Here's some tl;dr about GUI stuff after having a discussion about this ticket.

I don't see anything inherently wrong with the add friend screen. qTox, a Tox client, connecting to the Tox network, is asking for a Tox ID on the "add friends" page under the "add a friend" tab. With everything that has any kind of interface (a keyboard, a car, a piece of software) there are some things that are understood, and if they are somehow not known by the user, they are often reasonably simple to deduce. A user is expected to be ready to learn any time they approach something new. More or less, when you open software for the first time, it is understood that you're going to be seeing a screen you've never seen before and you're going to have to click around and figure out a few things before you get started. If the user can't deduce what a Tox ID is after having downloaded a Tox client, I'm not sure how much help can be given that wouldn't end up being a "nag" or annoyance for other users.

However this isn't to say that the GUI is perfect (it's not) and there are things that can be improved. The ability to quickly learn a new GUI in my experience is usually dependent on consistency. Consistency is what makes learning simple and quick. As soon as there are rule exceptions, things can get fucky fast as you have to memorize odd or random quirks. qTox is not entirely consistent and I actually have some GUI changes planned that are entirely focused on consistency, but I'm adding two more. The options menus should be in one spot: the toolbar in the bottom left. Finding your profile page, getting your Tox ID, and learning what the Tox ID is should take a matter of seconds for the average user. The top left (where you access your profile) is also inconsistent. Clicking your status changes your status, so therefore clicking your name should change your name and clicking your avatar should change your avatar. Another thing to consider is organization, or figuring out where things go or fit logically. Information about your profile should probably stay on your profile screen. I strongly urge against moving the user's Tox ID anywhere else. Finally, there's intuitiveness. It's not entirely intuitive (and even less consistent) that the profile screen is accessed via the top left by clicking name or avatar, but it's also not entirely illogical. qTox's absolute worst and most notorious example of unintuitiveness is the delete avatar option. Try to find it, good luck.

I think issues like these need to be approached with a heavy focus on consistency and a strong awareness of organization. Disorganizing the GUI to create a "start screen" of sorts with a bunch of "starter information" for example would only serve to impede usability in the long run as things are placed illogically. Things should be placed logically but in a consistent manner so that learning is fast.

tl;dr: fuckin' GUIs, how do they work?

**Comment by zUhj475tHG** [comment]
GUI-Idea 

The Tabs stay bottom left.

First Tab is: People. 
For this Tab there are 3 Tabs on the right side off the screen:
Tab 1: Add Friends & Import Contacts
Tab 2: Friend requests
Tab 3: Your Profile (Modify)

Second Tab is: Groups.
For this Tab there are 3 Tabs on the right side off the screen:
Tab1: Add Groups
Tab2: Group requests
Tab3: Your Groups (Create and Modify)

Top-left clicking on name, status and image should bring you all into "Your Profile". Must be the same behaviour for all of these I think, too.


Explaining:

The People Tab -> Change Icon from Plus to one Person

In the Add Friends & Import Contacts - Tab it would be really useful to give a step-by-step help how it works, like this: (could be a help popup, that can be de/activated)

how to add a friend:
    • receive your friends Tox ID.
    • add his TOX ID at "Add Friends"
    • he has to accept the requests at "Friend requests"

how to be added by a friend:
    • send your TOX ID to your Friend.
    • he adds your TOX ID at "Add Friends"
    • you accept the request at "Friend requests"

This way, the user starts at the Tab "Add Friend ..." and would have a nice explanation how to use qtox.
But where does he now find his own TOX ID that was mentioned?
    •  you could make at "send your TOX ID to ..." the "TOX ID" a clickable and colourful thing, that leads to "Your Profile"
    •  you can write "send your TOX ID to ... ." and add: (TOX ID to find at "Your Profile"). If the user is then searching for "Your Profile" - he is just two Tabs away from it. 
    • Both options combined should also be possible :)

The "Add groups" should work similar to the Add Friends Tab - you can insert an ID and connect with nice explanation. Additionally it would be nice if you have some general public groups (DHT based groups) that are offered in a list with their ID and Name. If you are interested in one of them or you are a new user, that not have other persons at qtox right now and wants to test it a bit -  they can simply click connect.
At "Your Groups" the user could create a DHT based Group and change groups options later on.
